src/ESP32-HUB75-MatrixPanel-I2S-DMA.h

```diff
@@ -764,40 +776,6 @@ class MatrixPanel_I2S_DMA {
    // ------- PRIVATE -------
   private:
 
-    // Matrix i2s settings
```

Comment:
Why have you decided to move all this stuff to public? Can we keep it private. I would prefer if people don't start changing these things on the fly through the class pointer and then raise issues on the back of that.